pybossa/cache/helpers.py

```diff
@@ -235,3 +237,21 @@ def n_locked_tasks(project_id):
     from pybossa.redis_lock import get_active_user_count
 
     return get_active_user_count(project_id, sentinel.master)
+
+
+def n_priority_x_tasks(project_id, priority=1.0, include_gold_task=False):
+    """Return the number of tasks with a given priority for a given project."""
+    if include_gold_task:
+        query = text('''SELECT COUNT(*) AS n_tasks FROM task
+                        WHERE project_id=:project_id AND state !='completed'
```

Comment:
state can be searched here and at line 250 as 
```
state = 'ongoing'
```
also, docstring can be updated to include ongoing tasks

Reply:
Thanks Deep. I think "ongoing" is clearer and less error pone than "reversed completed". Will update it. Curious, does technically our system treat not 'completed' as 'ongoing'? If there are some others states like 'error' then the answer would be no.

Reply:
thats correct, tasks not completed are marked as ongoing.